styleguide/_themes/global/scss/modifiers.scss

```diff
@@ -76,6 +76,18 @@ img { max-width: 100%; }
   margin: 0 auto;
 }
 
+// Horizontal rules
+hr {
```

Comment:
I've got it as a classed as hr-red and it remains and hr. 
To further address the feedback I think I would rather treat the disease and not the symptom and get a list of the unclassed hrs and either see why they are there or put them into their own maintainable classes if we think its a good idea. 
I know its a little more work than not adjusting the crazy margins on the hr in general but I think it's ultimately the best option.
@ejchiefs @WidgetsBurritos 

Up to the group.